react/PlaceOrder.css

```diff
@@ -0,0 +1,3 @@
+.buttonWrapper :global(.vtex-button) {
+  font-size: 1rem !important;
```

Comment:
❗️ 👀

Reply:
I talked with @davicosta99 and we will probably adjust this in styleguide :eyes: